cron/build_chrom_db.py

```diff
@@ -57,15 +61,19 @@ def getchrominfo(url, db):
             sys.exit("Bad input file.")
     else:
         try:
-            for build in parse_builds.getbuilds("http://genome-test.cse.ucsc.edu/cgi-bin/das/dsn"):
+            for build in parse_builds.getbuilds("http://genome.cse.ucsc.edu/cgi-bin/das/dsn"):
```

Comment:
The reason is that several of the builds downloaded by `parse_builds.getbuilds()` when using this URL will generate an error when downloading the len file. In fact, the default URL in `parse_builds.py` is the URL I'm using here.

I've no specific reason to change the URL below, maybe test is used because of download limits on stable.